src/server/server.js

```diff
@@ -19,22 +19,49 @@ import downloadRouter from './download';
 import CodedError from './utils/error';
 import { statusRouter, versionRouter } from './endpoints';
 
+let server;
 const app = express();
 app.use(cors());
 app.use(helmet());
 app.use(bodyParser.json({ limit: '50mb' }));
 
+const refreshRouter = async (req, res, next) => {
```

Comment:
`refreshRouter` should be placed after `initializeAndStartServer`

Reply:
fxied